src/collection/components/modals/CutFragmentModal.tsx

```diff
@@ -89,8 +95,10 @@ const CutFragmentModal: FunctionComponent<CutFragmentModalProps> = ({
 		const start = toSeconds(fragmentStartTimeString, true);
 		const end = toSeconds(fragmentEndTimeString, true);
 
-		updateFragmentProperty(start, 'start_oc', fragment.id);
-		updateFragmentProperty(end, 'end_oc', fragment.id);
+		updateFragmentProperties([
+			{ value: start, fieldName: 'start_oc' as const, fragmentId: fragment.id },
```

Comment:
Waarom hier die `as const`? (Heb dat nog niet eerder gezien 😄)

Reply:
omdat typescript anders denkt dat dat een generic string kan zijn. Door te zeggen dat het een const is, kan hij dat type ('start_oc') resolven met het type: (keyof Avo.Collection.Fragment)

Toegevoegd aan typescript 3.4: https://devblogs.microsoft.com/typescript/announcing-typescript-3-4-rc/
Onder de titel: const assertions